providers/packetframe/packetframeProvider.go

```diff
@@ -0,0 +1,237 @@
+package packetframe
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+
+	"github.com/StackExchange/dnscontrol/v3/models"
+	"github.com/StackExchange/dnscontrol/v3/pkg/diff"
+	"github.com/StackExchange/dnscontrol/v3/providers"
+)
+
+// packetframeProvider is the handle for this provider.
+type packetframeProvider struct {
+	client      *http.Client
+	baseURL     *url.URL
+	token       string
+	domainIndex map[string]zone
+}
+
+var defaultNameServerNames = []string{
+	"ns1v4.packetframe.com",
```

Comment:
These are beta nameservers; they should be `ns1.packetframe.com` and `ns2.packetframe.com` instead.

Reply:
Good catch!

Reply:
:+1: Also, we shouldn't add the nameservers as NS records to the zone - they're added in the backend by Packetframe.